src/main/java/pl/kruczala/michal/Casket200.java

```diff
@@ -0,0 +1,25 @@
+package pl.kruczala.michal;
+
+public class Casket200 extends CashMachine {
+
+    //int accountBalance = getAccountBalance();
+    int numberOf200BanknotesInCasket = 100;
+    int usedNumberOf200BanknotesToPayOut =-1;
+
+
+    public int casketPayOut200(int requestedSumInMain) {
+
+        requestedSumInMain = requestedSumInMain+ 200;
+
+        do {
+            requestedSumInMain = requestedSumInMain - 200;
+            numberOf200BanknotesInCasket = numberOf200BanknotesInCasket - 1;
+            usedNumberOf200BanknotesToPayOut = usedNumberOf200BanknotesToPayOut + 1;
```

Comment:
To powinna być zmienna wewnątrz funkcji ponieważ przy kolejnym wywołaniu dostanę sumę dwóch wywołań 

```
Casket200 casket = new Casket200();
casket.casketPayOut200(200);// zwróci jeden banknot
casket.casketPayOut200(200); //zwróci dwa banknoty
```